client/src/components/Toasts/Toasts.scss

```
@@ -0,0 +1,157 @@
+.toasts {
+  position: fixed;
+  top: 60px;
+  right: 20px;
+  width: 250px;
+  z-index: 9999;
+
+  &__notice-wrap {
+    position: absolute;
+  }
+}
+
+
+@keyframes toastFadeIn {
+  0% { opacity: 0; }
+  100% { opacity: 1; }
+}
+
+@keyframes toastFadeOut {
+  0% {
+    opacity: 1;
+    max-height: 250px;
+    margin-bottom: map_get($spacers, sm);
+  }
+
+  50% {
+    opacity: 0;
+    max-height: 250px;
+    margin-bottom: map_get($spacers, sm);
+  }
+
+  100% {
+    opacity: 0;
+    max-height: 0;
+    margin-bottom: 0;
+  }
+}
+
+
+.toast {
+  margin: 0 0 map_get($spacers, sm) 0;
+  border-radius: 3px;
+  font-family: inherit;
+  font-size: inherit;
+  word-wrap: break-word;
+  max-height: 250px;
+  border: 0;
+  border-left: 3px solid;
+  background: #fff;
+  opacity: 1;
+  box-shadow: $z-depth-1;
+  transition: box-shadow $transition-speed-fast ease;
+  width: 250px;
+
+  &:not(:last-child) {
+    margin-bottom: map_get($spacers, sm);
+  }
+
+  &:hover {
+    box-shadow: $z-depth-2;
+  }
+  animation: $transition-speed-mid ease-out 0s 1 toastFadeIn;
+
+  &--success,
+  &--good,
+  &--green {
+    border-color: map_get($theme-colors, success);
+  }
+
+  &--notice,
+  &--info,
+  &--blue {
+    border-color: map_get($theme-colors, info);
```

Will stick with Paul's initial designs for colour for now. Easy to change later. It's not a theme colour but is under $colors so assume it'd be the below? Or even just $cyan?

```suggestion
    border-color: map_get($colors, cyan);
```